From 44df6ff1a288c880d2848746ea043272145a5f76 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 15 Apr 2020 13:19:35 -0700
Subject: [PATCH] Don't enforce version matches

Version equality is too strict for maintaining this package so ignore
them.
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 0644dd5..64ea041 100644
--- a/setup.py
+++ b/setup.py
@@ -41,12 +41,12 @@ setup(
     zip_safe=False,
     python_requires=">=3.6",
     install_requires=[
-        "gevent==1.5a3",
+        "gevent",
         "flask>=0.10.1", 
         "requests>=2.9.1", 
         "msgpack>=0.6.2", 
         "pyzmq>=16.0.2", 
-        "geventhttpclient-wheels==1.3.1.dev2",
+        "geventhttpclient-wheels",
         "ConfigArgParse>=1.0",
         "psutil>=5.6.7",
     ],
-- 
2.26.0

